packages/tools/devtools/devtools-view/package.json

```diff
@@ -12,14 +12,17 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
```

Comment:
To get a full understanding of all the changes here... this is so we can import the stories tsx files as modules even though `export default` does not indicate that they are? And the new webpack config rules (`fullySpecified: false`) are so we're not forced to use extensions when importing them, or importing things inside them? The bit about the extensions in imports I thought was only relevant if we were compiling as ESM, which AFAICT we're not doing for this package.

Reply:
Ah, right. I forgot to bring this up in the meeting. This change is specifically because the @previewjs packages are ESM only. Rather than try to force compatibility with this as a CommonJS library, I went ahead and just moved to ESM for this package. Since it's (for now) strictly an implementation detail of the extension, it seemed a reasonable path forward, and managed to unblock me.

It is not specifically related to the story format, since we don't actually compile those modules ourselves.

Reply:
(I added notes to the PR description about this)